src/library.rs

```diff
@@ -192,20 +215,25 @@ pub struct Alias {
     pub c_identifier: String,
     pub typ: TypeId,
     pub target_c_type: String,
+    pub doc: Option<String>,
```

Comment:
For now I have set deprecated doc to "========> DEPRECATED <=========". Should be enough. :p